### PR TITLE
(DOCSP-34626): Realm Studio: New View Atlas Data with Device Sync page

### PR DESCRIPTION
**Pull request by dacharyc**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-34626

Staged changes:

- [View Data with Device Sync](https://preview-mongodbdacharyc.gatsbyjs.io/realm/DOCSP-34626/studio/view-atlas-data-sync/): New page describing the new functionality and how to use it.
- [Realm Studio landing page](https://preview-mongodbdacharyc.gatsbyjs.io/realm/DOCSP-34626/studio/): Tweak the intro to mention the new functionality and reword "create Realm files."

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for corresponding docs-realm docs-app-services, if any

### Release Notes

- **Realm Studio**
  - Landing Page: Add a bullet listing the ability to connect, view, and modify data using Device Sync.
  - View Data with Device Sync: New page describing how to use Realm Studio to test subscriptions and view Atlas data changes in real time with Device Sync.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/studio/view-atlas-data-sync.txt [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+            TRUEPREDICATE
+
+   .. step:: Add the subscription
+
+      Press the :guilabel:`Add subscription` button to complete the process
+      of adding a subscription. This triggers an initial sync with Device Sync
+      for:
+
+      - Items of the selected object type.
```
[the remark this answers is not in the document]
Comment (cbullinger) [reply source/studio/view-atlas-data-sync.txt:182]: Looking at it on the staged page doesn't bug me as much. maybe ignore this

File: source/studio/view-atlas-data-sync.txt [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+Realm Studio can connect via Device Sync using any of these authentication
+provider types:
+
+- Anonymous
+- Email & Password
+- API Key
+- JWT
+
+If your app uses Email & Password, API Key, or JWT authentication, you must
```
Comment (cbullinger) [inline source/studio/view-atlas-data-sync.txt:47]: capital A App?

File: source/studio/view-atlas-data-sync.txt [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+      After you have successfully connected, Realm Studio shows the App ID
+      in the menu bar of the application window. You can open connections to 
+      multiple App Services Apps simultaneously. You can also open multiple
+      connections to the same App Services App if you want to view changes
+      in real time.
+
+Specify the Data to Sync
+------------------------
+
+After you connect to Atlas from Realm Studio, the app displays a list of
```
Comment (cbullinger) [inline source/studio/view-atlas-data-sync.txt:89]: `the app displays` -- too many apps... which app is this?

File: source/studio/view-atlas-data-sync.txt [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+Connect to Atlas from Realm Studio
+----------------------------------
+
+.. versionadded:: 14.1.2
+
+.. procedure:: 
+
+   .. step:: Select Connect via Atlas Device Sync
+      
+      Open Realm Studio, and press the
```
[the remark this answers is not in the document]
Comment (dacharyc) [reply source/studio/view-atlas-data-sync.txt]: Hah, which just goes to show the last time I wrote a lot of UI docs was... 2017? 😁

**Pull request merged**